### ISSUE TITLE
Is there any plan to reopen the support of the old-fashion zotfile-style "extract annotations"?

### ISSUE THREAD

**Issue by yangfurong** [jlegewie/zotfile#591]
Hi. First of all, I would like to thank you for the development and maintenance of Zotfile. 

However, I am currently facing a problem with the integrated new function "Add Note from Annotations" in Zotero 6.0. 

In Zotero 5.0, I use Zotfile to extract annotations from PDF files to get notes. In those old-fashion notes, for every annotation item, there is a link based on zoteroURI (e.g. "Lee et al 2013:536" links to zotero://open-pdf/library/items/2LW4SP7I?page=1), which allows us to jump into a PDF file at a given location. In the past, I usually export my notes to a dedicated notes management software, Obsidian (i.e. the workflow is like this, https://bagerbach.com/blog/how-i-read-research-papers-with-obsidian-and-zotero). Thus, those zoteroURI based links are vital for me, as they allow me to jump into PDF files at given locations from Obsidian directly. But, in Zotero 6.0, the notes generated by "Add Note from Annotations" no more have explicit zoteroURI-based links, which blocks me from opening PDF files at given locations in Obsidian.

**Comment by dstillman** [comment]
Yes, Zotero 6 includes built-in support for Markdown export via Quick Copy, with `zotero://open-pdf` links back to the PDF. You can test with any plain-text editor. With Obsidian, you have to change a setting to prevent it from stripping those links:

https://forums.zotero.org/discussion/comment/401954#Comment_401954

**Comment by yangfurong** [comment]
Thanks for your comments. It solves the problem.
